enterprise/apps.py

```diff
@@ -17,7 +18,7 @@ class EnterpriseConfig(AppConfig):
 
     name = "enterprise"
     valid_extensions = [".png", ]
-    image_size = 250 * 1024
+    image_size = getattr(settings, 'EC_LOGO_IMAGE_SIZE', 4 * 1024)
```

Comment:
Nit: `ENTERPRISE_CUSTOMER_LOGO_IMAGE_SIZE`: we're not short on bytes or line space; there's no need to abbreviate. This is going to be a variable that's going to be mixed in with variables for every other  app; we need it to be distinctive.